ghost/core/core/server/api/endpoints/utils/serializers/output/utils/extra-attrs.js

```
@@ -8,31 +8,35 @@ const readingMinutes = require('@tryghost/helpers').utils.readingMinutes;
+                attrs.excerpt = plaintext.substring(0, 500);
             } else {
-                attrs.excerpt = attrs.custom_excerpt;
-                if (!options.columns?.includes('custom_excerpt')) {
-                    delete attrs.custom_excerpt;
-                }
+                attrs.excerpt = null;
+            }
+        } else {
+            attrs.excerpt = attrs.custom_excerpt;
+            if (!columnsIncludesCustomExcerpt) {
+                delete attrs.custom_excerpt;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)